### ISSUE TITLE
[Feature request] Add support for 1.12.2

### ISSUE THREAD

**Issue by TrevorBagels** [djeezuss/Valkyrien-Warfare-Drivers-for-OpenComputers#2]
Most people I know use 1.12 rather than 1.11, so a version of the mod for 1.12 would be nice.

**Comment by TechTastic** [comment]
Agreed but this mod hasn't had an update (on CurseForge) in 3-4 years so sadly I think it might be abandoned

**Comment by TrevorBagels** [comment]
Yep, it appears to be that way. Rip.
